gcc/config/arc/arc.md

```diff
@@ -3978,6 +4096,45 @@
    (set_attr "iscompact" "true")
    (set_attr "cond" "nocond")])
 
+(define_insn "jli_s"
+  [(match_operand:SI 0 "immediate_operand" "L")]
+  ""
+  "jli_s %0"
+  [(set_attr "type" "call")
+  (set_attr "iscompact" "true")
+  (set_attr "predicable" "yes")
```

Comment:
Probably you want to say "predicable" "no" as the instruction string doesn't print any predication.
 Add a line note